### ISSUE TITLE
Gaussian distribution  (--key-pattern=G:G) stalls memtier on macOS (at least)

### ISSUE THREAD

**Issue by filipecosta90** [RedisLabs/memtier_benchmark#98]
hi there, running memtier with the Gaussian distribution option on macOS is stalling it ( build with TLS support is disabled  ).
```
memtier_benchmark --hide-histogram  --key-pattern=G:G
```
Output:
```
$ memtier_benchmark --hide-histogram  --key-pattern=G:G 
[RUN #1] Preparing benchmark client...
[RUN #1] Launching threads now...
[RUN #1 0%,   0 secs]  4 threads:           0 ops,       0 (avg:       0) ops/sec, 0.00KB/sec (avg: 0.00KB/sec),   nan (avg:   nan) msec latency
```

**Comment by QingchenDang** [comment]
Same problem. Does anyone know how to fix it?

**Comment by yossigo** [comment]
@YaacovHazan Duplicate of #158?

**Comment by YaacovHazan** [comment]
Closed, duplicate of #158
